source/Controlled.tsx

```diff
@@ -73,7 +75,11 @@ const Controlled: FC<Props> = ({
 
   return (
     <StrictMode>
-      <div data-rmiz-wrap={wrapType} ref={wrapRef} style={wrapStyle}>
+      <WrapElement
+        data-rmiz-wrap={wrapType}
+        ref={wrapRef as React.RefObject<HTMLDivElement>}
```

Comment:
1. `RefObject` can be destructured off of `React` at the top
1. does this not limit it to being only `HTMLDivElement`?

Reply:
> does this not limit it to being only HTMLDivElement?

Good point! Somehow the compiler didn't catch that one.